src/utils/sosmeta-utils.ts

```diff
@@ -160,9 +172,13 @@ namespace SosmetaUtils {
    * @returns MetaformField
    */
   const handleSosmetaBooleanSection = (sosmetaSection: any) => {
-    const { required } = sosmetaSection;
+    try {
+      const { required } = sosmetaSection;
 
-    return [ handleSosmetaBooleanField(sosmetaSection.sosmeta.name[0].value, sosmetaSection, required) ];
+      return [ handleSosmetaBooleanField(sosmetaSection.sosmeta.name[0].value, sosmetaSection, required) ];
+    } catch (e) {
+      // TODO: Error handling/messagess
```

Comment:
Maybe error handling could be just console.error(e) in these cases.

Reply:
Done, for all sosmeta utils. Now its clearer that there are some errors when we translate certain sosmeta forms, but this is working as before.